File: TPs/TP1/TP01Q10.c
#include <string.h>
#include <stdbool.h>
#include <stdio.h>

int main()
{
	char palavra[100];
	fgets(palavra, sizeof(palavra),stdin);
	int contagem = 0;
	int tamanho = strlen(palavra);


	for(int i = 0; i < tamanho; i++) {

		palavra[i] = palavra[i] + 3;

		if(palavra[i] == '#') {
			contagem++;
		}

	}
	contagem++;

	printf("%d", contagem);

	return 0;
}

File: TPs/TP1/TP01Q10.java
public class TP01Q10
{
	public static void main(String[] args) {
	    
	    boolean isfim = false;
	    do{
	    
	    String palavra = MyIO.readLine();
	    char[] vet = palavra.toCharArray();
	    int contagem = 1;
	    int tamanho = vet.length;
	    
	    if(palavra.equalsIgnoreCase("FIM")){ // Função para caso a palavra lida for igual a FIM
	        isfim = true;
	        break;
	    }
	    
	    for(int i = 0; i < tamanho; i++){
	        
	        if(vet[i] == ' '){
	            contagem++;
	        }
	    }
	     
	    MyIO.print(contagem);
	    MyIO.println(" ");
	    
	    }while(isfim != true);
		
	}
}



